### ISSUE TITLE
Task stuck in uploading state after power outage

### ISSUE THREAD

**Issue by sirzooro** [BOINC/boinc#1700]
I've just had an unexpected power outage. After recovering from it I found than one of SRBase tasks is stuck in "uploading" state, and there are no files listed on Transfers tab. I tried to abort it but this did not have any effect.

How can I remove this task, or somehow unstuck it?

Boinc 7.7.0, Linux CentOS 7

Edit: I removed project and added it again using new URL as I saw in logs, and after doing this that task is gone. However BOINC client should check for such stuck tasks during startup and do something with them.

**Comment by sirzooro** [comment]
I have got this issue one more time. This is on another machine which is connected to UPS, so there was no power outage there. I suspect that this happened when I restarted machine in order to load new kernel after upgrade. This is also on CentOS 7, BOINC 7.6.22.

**Comment by ChristianBeer** [comment]
This was reported on Einstein@Home too not long ago. It seems there is a bug when uploading is interrupted at the wrong time. The user changed the state directly in `client_state.xml` and that cleared the task but this is only for advanced users. Someone needs to investigate how to make the client aware of such stuck tasks and retry upload again.

I see this was classified as a feature request by me which seems wrong. I'm fixing that but don't have time to troubleshoot the actual bug.

**Comment by smoe** [comment]
This just happened to me. First time ever. The task is [LATeah1049ag_188.0_0_0.0_45710406](https://einsteinathome.org/de/task/881758481). Already aborted it but that info was not forwarded to the project and had no other effect.
The task is still present after a stop and start of the boinc-client. But that restart has apparently freed the slot so I got all GPUs active again.

Ubuntu Bionic. BOINC 7.16.1.

**Comment by smoe** [comment]
I found this information
```
Sep 11 12:51:07 rig1c boinc[1024]: 11-Sep-2019 12:51:07 [Einstein@Home] File projects/einstein.phys.uwm.edu/LATeah1049ag_188.0_0_0.0_45710406_0_0 has wrong size: expected 415, got 0
Sep 11 12:51:07 rig1c boinc[1024]: 11-Sep-2019 12:51:07 [Einstein@Home] File projects/einstein.phys.uwm.edu/LATeah1049ag_188.0_0_0.0_45710406_0_1 has wrong size: expected 269, got 0
```
in /var/log/syslog but not among the events.

Also, the slot apparently got reoccupied by the upload.

**Comment by davidpanderson** [comment]
It looks like this was fixed in 5774beafb68
